romancal/datamodels/schemas/core.schema.yaml

```diff
@@ -96,71 +84,56 @@ properties:
           elapsed_exposure_time:
             title: "[s] Total elapsed exposure time"
             type: number
-#          nints:
-#            title: Number of integrations in exposure
-#            type: integer
-#          integration_start:
-#            title: Starting integration number in this segment
-#            type: integer
-#          integration_end:
-#            title: Ending integration number in this segment
-#            type: integer
-#          frame_divisor:
-#            title: Divisor applied to frame-averaged groups
-#            type: integer
-#          groupgap:
-#            title: Number of frames dropped between groups
-#            type: integer
-#          drop_frames1:
-#            title: Frames dropped at start of each integration
-#            type: integer
-#          drop_frames3:
-#            title: Frames dropped at end of each integration
-#            type: integer
-#          nsamples:
-#            title: Number of A/D samples per pixel
-#            type: integer
-#          sample_time:
-#            title: "[us] Time between samples"
-#            type: number
-#          frame_time:
-#            title: "[s] Time between frames"
-#            type: number
-#          group_time:
-#            title: "[s] Time between groups"
-#            type: number
-#          exposure_time:
-#            title: "[s] Effective exposure time"
-#            type: number
-#          effective_exposure_time:
-#            title: "[s] Effective exposure time"
-#            type: number
-#          duration:
-#            title: "[s] Total duration of exposure"
-#            type: number
-#          nresets_at_start:
-#            title: Number of resets at start of exposure
-#            type: integer
+          nints:
```

Comment:
Since there is always just a single integration, why are there keywords pertaining to integrations such as 'nints' and 'integration_start' ? For compatibility with JWST code ? ( Apologies if this has been discussed/noted elsewhere already )

Reply:
1. Typo fixed
2. & 3. were discussed and dealt with in our offline meeting.

Reply:
The integration keywords remain in case a large volume of code depends upon them, according to our Level1 document.